types/schemas/preApplication/enums/PreApplicationTypes.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Values of `data.application.type`
+ */
+export const PreApplicationTypes = {
+  preApp: 'Pre-application advice',
+  'preApp.householder': 'Pre-application advice - Householder',
+  'preApp.minor': 'Pre-application advice - Minor',
+  'preApp.major': 'Pre-application advice - Major',
```

Comment:
We're just using the one type for now (`preApp`) but it's possible there may be others later so this is more an indication of intent